### PR TITLE
Fix uitests

### PR DESCRIPTION
**Pull request by KamaldeepSekhon**

### CODE REVIEW

File: cypress/e2e/signup.cy.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ describe('signup', () => {
         cy.contains('Sign up').should('exist');
         cy.get('input#firstName').should('exist').type('Kamal');
         cy.get('input#lastName').should('exist').type('Sekhon');
-        cy.get('input#email').should('exist').type('kks@gmail.com');
+        cy.get('input#email').should('exist').type('kamalks@gmail.com');
```
Comment (nikojohnarellano) [inline cypress/e2e/signup.cy.ts:12]: I already have a fix for this. no need to change this

File: cypress/e2e/createappointment.cy.ts [hunk 2]
```diff
@@ -15,31 +15,46 @@ describe('Create New Appointment', () => {
...
+                const del = 'button[type="button"]>svg';
+                cy.get('[class*="css-t1nuxs"]')
+                    .first()
+                    .then(($x) => {
+                        const data1 = $x.text();
+                        cy.get(del).first().click({ multiple: true });
+                        // check if the appointment is deleted or not
+                        cy.get('[class*="css-t1nuxs"]').first().contains(data1).should('not.exist');
+                    });
+            }
```
Comment (nikojohnarellano) [inline cypress/e2e/createappointment.cy.ts:56]: this is unnecessarily complicated and it is also prone to breakages. maybe we can do this another way? 

I looked at the html element. Seems like we can use the "DeleteIcon" test id. 

```
const APPOINTMENT_TITLE = 'help with gardening';

cy.contains(APPOINTMENT_TITLE).parent().find('[data-testid="DeleteIcon"]').click()
```

File: cypress/e2e/createappointment.cy.ts [hunk 3]
```diff
@@ -15,31 +15,46 @@ describe('Create New Appointment', () => {
...
+            .clear()
+            .type('11/29/2022 03:00 PM');
+
+        cy.contains('button', 'Schedule').should('exist').click({ multiple: true });
         cy.contains('Appointments').should('exist');
-        cy.wait(2000);
+
         // Check if the appointment details are displayed on the list
-        cy.contains('help with laundry').should('exist');
+        cy.contains('help with gardening').should('exist');
```
Comment (nikojohnarellano) [inline cypress/e2e/createappointment.cy.ts:36]: Is this working? the appointment titles are inconsistent. You are typing "help with groceries" above and now you have "help with gardening"

Comment (nikojohnarellano) [reply cypress/e2e/createappointment.cy.ts:36]: Can we extract this one into a reusable constant

File: cypress/e2e/createappointment.cy.ts [hunk 4]
```diff
@@ -15,31 +15,46 @@ describe('Create New Appointment', () => {
...
-            .get('div.MuiPaper-root')
-            .click()
-            .get('.MuiClock-squareMask')
-            .click();
-        // cy.wait(2000);
-        cy.contains('button', 'Schedule').should('exist').click();
+            .get('li.MuiButtonBase-root:nth-child(10)')
+            .click({ multiple: true });
+        const timeNow = new Date().getTime();
+        cy.get('input#title').should('exist').type('help with groceries ').type(timeNow.toString());
```
Comment (nikojohnarellano) [inline cypress/e2e/createappointment.cy.ts:24]: see comment below

**Review by zigfrida — DISMISSED** [review]

**Review by nikojohnarellano — APPROVED** [review]

**Pull request merged**